contracts/test/AllowanceTarget/CompleteSetSpender.t.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.6;
+
+import "contracts-test/AllowanceTarget/Setup.t.sol";
+
+contract TestAllowanceTargetCompleteSetSpender is TestAllowanceTarget {
+    function testCannotSetBeforeSetNew() public {
```

Comment:
Would it be more smooth to name by `complete` instead of `set`? For example, `testCannotCompleteBeforeSet `